internal/pool/conn_check.go

```diff
@@ -0,0 +1,49 @@
+// +build linux darwin dragonfly freebsd netbsd openbsd solaris illumos
+
+package pool
+
+import (
+	"errors"
+	"io"
+	"net"
+	"syscall"
+	"time"
+)
+
+var errUnexpectedRead = errors.New("unexpected read from socket")
+
+func connCheck(conn net.Conn) error {
+	//reset deadline
+	_ = conn.SetDeadline(time.Time{})
```

Comment:
@naiqianz could you fix the comment formatting?

```go
// Reset previous timeout.
```

Reply:
sure, tks for ur suggestion